mongodump/oplog_dump.go

```diff
@@ -86,7 +85,7 @@ func (dump *MongoDump) checkOplogTimestampExists(ts bson.MongoTimestamp) (bool,
 }
 
 func oplogDocumentFilter(in []byte) ([]byte, error) {
-	var rawD bson.RawD
+	var rawD bson.D
```

Comment:
Nit: the variable name doesn't match the type anymore because we're using a bson.D instead of bson.Raw.

Reply:
Removed as part of changes re: your other comment.